src/config.rs

```diff
@@ -68,14 +68,14 @@ pub struct BuilderConfig {
         var = "HOST_RPC_URL",
         desc = "URL for Host RPC node. This MUST be a valid HTTP or WS URL, starting with http://, https://, ws:// or wss://"
     )]
-    pub host_rpc_url: url::Url,
+    pub host_rpc: init4_bin_base::utils::provider::ProviderConfig,
```

Comment:
is there any reason we're using the entire path instead of just importing the used structs?